homeassistant/data_entry_flow.py

```diff
@@ -175,6 +164,28 @@ def async_abort(self, *, reason):
         return {
             'type': RESULT_TYPE_ABORT,
             'flow_id': self.flow_id,
-            'domain': self.domain,
+            'handler': self.handler,
             'reason': reason
         }
+
+
+class SingleSchemaFlow(FlowHandler):
```

Comment:
Maybe should be inside `helpers/flows.py`?

Reply:
Good point, I didn't want to include this one yet. It's for the auth work. Let me remove it.